roles/confluent.variables/defaults/main.yml

```diff
@@ -291,45 +291,48 @@ zookeeper_custom_properties: "{{ zookeeper.properties }}"
 
 
 #### Kafka Broker Variables ####
-kafka_broker_default_listeners:
-  internal:
-    name: INTERNAL
-    port: 9092
-    ssl_enabled: "{{ ssl_enabled }}"
-    ssl_mutual_auth_enabled: "{{ ssl_mutual_auth_enabled }}"
-    sasl_protocol: "{{ 'OAUTH' if rbac_enabled|bool else sasl_protocol }}"
-
-kafka_broker_custom_listeners:
-  broker:
-    name: BROKER
-    port: 9091
-    ssl_enabled: "{{ ssl_enabled }}"
-    ssl_mutual_auth_enabled: "{{ ssl_mutual_auth_enabled }}"
-    sasl_protocol: "{{ sasl_protocol }}"
-
-kafka_broker_control_plane_listener:
-  controller:
-    name: CONTROLLER
-    port: 9089
-    ssl_enabled: "{{ ssl_enabled }}"
-    ssl_mutual_auth_enabled: "{{ ssl_mutual_auth_enabled }}"
-    sasl_protocol: "{{ sasl_protocol }}"
+
+kafka_broker_default_listeners: "{
```

Comment:
So initially `kafka_broker_custom_listeners` was the only one meant to be customized. and I kept it around in this pr:

`kafka_broker_custom_listeners: {}` 

my thought is it will be the only one users should be editing... but maybe i have not thought it thru fully.

So if you wanted to update the control plane listener port it would look like:

```
kafka_broker_configure_control_plane_listener: true
kafka_broker_custom_listeners:
  controller: 
    port: 9094
```

And that will get merged in

Reply:
Yes `kafka_broker_custom_listeners` should be the only one parameter that an user should really change in my opinion too. I'm just wondering if `kafka_broker_default_listerners` is customized in some corner case. I will change the default port for the control plane. It's late night here,  I will work on it tomorrow.

Reply:
You can choose that port, that was just an example.
Thanks for your work on this!

Reply:
Oh yes got it! it is because the combine filter will of course merge the values in the final variable. That should work out of the box. I will merge into my branch and modify the final pull request to the main repo